modal_proto/api.proto

```diff
@@ -2704,6 +2704,12 @@ message Sandbox {
 
   // Optional idle timeout in seconds. If set, the sandbox will be terminated after being idle for this duration.
   optional uint32 idle_timeout_secs = 33;
+
+  // Set only by the server. Must not be transmitted to the client. Used to
+  // authenticate the client when directly connecting to a sandbox server running on
+  // the modal-worker to issue exec commands (and other operations as they
+  // become available).
+  optional bytes sandbox_command_router_auth_secret = 34;
```

Comment:
I'm not really on board with making this symmetric secret put in the database and passed to every worker. IMO, the Sandbox definition should be something we can use in investigations, give to users, etc. It should not have secret information in it.

I think we should use our existing OIDC infrastructure to mint tokens and validate them from the Worker.